pandas/io/sas/sasreader.py

```diff
@@ -31,6 +31,9 @@ def read_sas(filepath_or_buffer, format=None, index=None, encoding=None,
     """
 
     if format is None:
+        bufferr = "Format unrecognized. If buffer object, specify format")
+        if type(filesize_or_buffer) != str:
```

Comment:
compat.string_types doesn't appear to be a standard library item? Instead using:
    type(filepath_or_buffer) not in [str, unicode]:
        raise TypeError(BuffErr)